lib/manageiq/loggers/journald.rb

```diff
@@ -8,7 +8,7 @@ class Journald < Base
       # progname option to 'manageiq' if not specified.
       #
       def initialize(_logdev = nil, *_, **_)
-        require "systemd-journal"
+        require "systemd-journal" if RbConfig::CONFIG['host_os'] =~ /linux/i
```

Comment:
Assume this is for mac when running the tests, we already have a `:linux` tag on the spec tests could you use that to skip instead?

Reply:
Weird, I'm taking a look.